test/asyncio_tests/test_asyncio_basic.py

```diff
@@ -151,6 +152,7 @@ def __getitem__(self, name):
 
 
 class ExecutorForkTest(AsyncIOTestCase):
+    @unittest.skipIf(sys.platform == "win32", "This test requires fork")
```

Comment:
It would be more robust to do: `hasattr(os, "fork)`

Reply:
Done.